### PR TITLE
Improve documentation more

### PR DESCRIPTION
**Pull request by GiudGiud**
Add parameter groups
Refs #20535

Add some missing stubs

Fixes some typo

Mastodon patch is ready
https://github.com/idaholab/mastodon/pull/430

### CODE REVIEW

**Comment by GiudGiud** [comment]
All failures should be unrelated now.
Need a volunteer for reviewing among all the code owners

**Comment by GiudGiud** [comment]
@joshuahansel can you please review?

**Review by joshuahansel — CHANGES_REQUESTED** [review]

File: modules/ray_tracing/src/userobjects/ConeRayStudy.C [hunk 1]
```diff
@@ -24,7 +24,7 @@ ConeRayStudy::validParams()
 
   params.addRequiredParam<std::vector<Point>>("start_points", "The point(s) of the cone(s).");
   params.addRequiredParam<std::vector<Point>>(
-      "directions", "The direcions(s) of the cone(s) (points down the center of each cone).");
+      "directions", "The directions(s) of the cone(s) (points down the center of each cone).");
```
Comment (GiudGiud) [inline modules/ray_tracing/src/userobjects/ConeRayStudy.C:27]:
```suggestion
      "directions", "The directions(s) of the cone(s) (points down the center of each cone).");
```
```suggestion
      "directions", "The direction(s) of the cone(s) (points down the center of each cone).");
```

File: modules/contact/doc/content/source/constraints/CartesianMortarMechanicalContact.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+## Overview
+
+It applies mortar generalized forces from Lagrange multipliers defined in the global Cartesian frame of reference.
+This can be used for both frictional and frictionless contact.
+
+## Example Input File Syntax
+
+In this example, we solve a cylinder-on-plane plane strain frictional problem with Cartesian Lagrange multipliers.
+Subdomain `10000` and `10001` are lower dimensional blocks (1D here) created from the `2` and `3` sidesets respectively,
+which are on both sides of the contact axis. c
```
Comment (GiudGiud) [inline modules/contact/doc/content/source/constraints/CartesianMortarMechanicalContact.md:14]:
```suggestion
which are on both sides of the contact axis.
```

**Review by joshuahansel — DISMISSED** [review]

**Comment by GiudGiud** [comment]
ugh lemme remove this merge commit

**Review by dschwen — DISMISSED** [review]
I like the new parameter groups. Just for future reference, we now have the ability to add doc strings to individual moose enum items, just in case you're looking for more stuff to document...

**Comment by GiudGiud** [comment]
looks like I ll have to wait for a devel merge before being able to address the merge conflict. 
That s ok though

**Comment by GiudGiud** [comment]
> Just for future reference, we now have the ability to add doc strings to individual moose enum items, just in case you're looking for more stuff to document...

tempting...

**Review by dschwen — APPROVED** [review]
re-approve

**Comment by GiudGiud** [comment]
Thanks!

**Pull request merged**